packages/loader/container-loader/src/utils.ts

```diff
@@ -424,3 +430,13 @@ export async function getDocumentAttributes(
 
 	return attributes;
 }
+
+/**
+ * Type guard to check if the given deltaManager is of type IDeltaManagerInternal.
```

Comment:
Nit
```suggestion
 * Type guard to check if the given deltaManager is of type {@link IDeltaManagerInternal}.
```

Reply:
Given that `IDeltaManagerInternal` lives in `container-definitions`, would it make more sense for this type-guard to live there as well?

Reply:
I think `container-definitions` should remain types only as best we can. (darn enums) That said there are a few helpers there already and I don't know where such a util should live, if not here or there.

Reply:
Yes, I am of the same opinion where I don't want to put helper functions into the definition's packages.

Reply:
Hmm. I guess a type-narrowing free function seems pretty appropriate for a types-focused package. I don't feel super strongly about this, but I do think we should avoid recreating the same function in a bunch of places. Is there a reasonable place we could share this function, if not in the same package as the type?

Reply:
Moved it to container-definitions